users/urls.py

```diff
@@ -4,4 +4,6 @@
 
 urlpatterns = [
     path('login/', auth_views.LoginView.as_view(template_name='users/login.html'), name='login'),
+    path('logout/', auth_views.LogoutView.as_view(template_name='users/logout.html'), name='logout'),
```

Comment:
maybe just redirect to homepage.

Reply:
It's a link between the three pages I did not understand what you are offering bro

Reply:
If you mean that the home page will be different it will make us problems later on and the design should be uniform. It is possible to make a base page in a different style from the landing page i think..